src/views/viewer/EmbeddedView/EmbeddedView.tsx

```diff
@@ -54,6 +54,8 @@ export const EmbeddedView: React.FC = () => {
     })
   }, [addVideoView, videoId, channelId, categoryId])
 
+  const allowFullscreen = (window.frameElement as HTMLIFrameElement)?.allowFullscreen
```

Comment:
I thought that it will be working just out of the box. We just pass `allowFullScreen`, and every element inside the iframe has fullscreen disabled.  Unfortunately, it won't work like that in our case. We need to pass the attribute and read it in the component. I could be wrong, but I think it's because we have several HTML elements in EmbededdedView. 

I've noticed that it only works if we had only a native video inside this iframe like in this snippet: 
```js
      <iframe
        src="http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/BigBuckBunny.mp4"
        frameBorder="2"
        width={600}
        height={400}
        allowFullScreen={false}
      ></iframe>
```